src/index.ts

```diff
@@ -94,7 +94,9 @@ export class Client {
     req.setModelname(modelName)
     const list: any[] = []
     values.forEach(v => {
-      v['ID'] = uuid.v4()
+      if (!v.hasOwnProperty('ID')) {
```

Comment:
Don't overwrite ID if it's an existing entity (probs leftover from copy-pasta).

Reply:
ahhh, I see now where this is coming from. See other comment about adding an empty ID if not there.